manifests/repo.pp

```diff
@@ -80,15 +80,15 @@
 
     'RedHat': {
       $enabled = $ensure ? { present => '1', absent => '0', default => absent, }
-      yumrepo { 'ext-epel-6.8':
+      yumrepo { 'ext-epel':
         # puppet versions prior to 3.5 do not support ensure, use enabled instead
         enabled    => $enabled,
-        descr      => 'External EPEL 6.8',
-        name       => 'ext-epel-6.8',
+        descr      => 'External EPEL',
+        name       => 'ext-epel',
         baseurl    => absent,
         gpgcheck   => '0',
         gpgkey     => absent,
-        mirrorlist => 'https://mirrors.fedoraproject.org/metalink?repo=epel-6&arch=$basearch',
+        mirrorlist => "https://mirrors.fedoraproject.org/metalink?repo=epel-${::operatingsystemmajrelease}&arch=\$basearch",
```

Comment:
it does not work on fedora, operatingsystemmajrelease is 21 for example and EPEL21 does not exist.